server_admin/topics/users/proc-creating-user.adoc

```diff
@@ -12,8 +12,8 @@ You create users in the realm where you intend to have applications needed by th
 * You are in a realm other than the master realm.
 
 .Procedure
-. In the admin console, click *Users* in the left menu.
-. Click *Add User* on the right side of the empty user list.
+. In the admin console, click *Users* in the main menu.
```

Comment:
Delete "In the admin console"  
Making sure you are not in the master realm implies you are already in the admin console.